tracks/r/exercises/sum-of-multiples/mentoring.md

```diff
@@ -1,29 +1,30 @@
-### Reasonable solutions
+### Reasonable solution
 
 This is a nice, concise, efficient solution:
 
 ```r
 sum_of_multiples <- function(factors, limit) {
-  func <- function(x) seq(0, limit - 1, x)
+  func <- function(x) { seq(0, limit - 1, x) }
   sum(unique(unlist(lapply(factors, func))))
 }
 ```
 
 
 ### Common suggestions
 
-- point learners who use a `for` loop to the alternative `apply` or `map` function families
-    + for example explained by one of these:
-        * https://statcompute.wordpress.com/2018/09/15/how-to-avoid-for-loop-in-r/
-        * https://r4ds.had.co.nz/iteration.html#for-loops-vs.functionals
-        * https://www.rstudio.com/resources/videos/happy-r-users-purrr-tutorial/
-        * https://www.datacamp.com/community/tutorials/tutorial-on-loops-in-r#the-alternatives-to-loops-in-r
-    + performance benefit esp. for larger input vectors
-    + the code's intention can be expressed better by a better `func` name
+- Point learners who use a `for` loop to the alternative `apply` or `map` function families
+    + For example, explained by one of these:
+        * [How to Avoid For Loop in R](https://statcompute.wordpress.com/2018/09/15/how-to-avoid-for-loop-in-r/)
+        * [For loops vs. functionals](https://r4ds.had.co.nz/iteration.html#for-loops-vs.functionals)
+        * [Happy R Users Purrr](https://www.rstudio.com/resources/videos/happy-r-users-purrr-tutorial/)
+        * [What is Vectorization?](https://www.datacamp.com/community/tutorials/tutorial-on-loops-in-r#the-alternatives-to-loops-in-r)
```

Comment:
Converting plain URLs to hyperlinks is _looks_ nicer, but _makes_ it harder to insert them into a comment to learners. At least my Firefox copies only the link text. Please wrap in ``` `backticks` ``` to make copy-pasting the whole link easy.

Reply:
Done so.

Reply:
With "whole" I meant it like in 063a525, sorry. How about that?